<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddResearch</class>
 <widget class="QWidget" name="AddResearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit research</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,1,1,1">
     <item row="4" column="2">
      <widget class="QPushButton" name="editResearchersButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QDateEdit" name="endDate">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="deletePointButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Checklist:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Start date:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Manage researchers:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="startDate">
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Information:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="5">
      <widget class="QTextEdit" name="researchInfo">
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="5">
      <widget class="QListWidget" name="checklist"/>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>End date:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="newPointButton">
       <property name="text">
        <string>New point</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="editPointButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLineEdit" name="researchTitle"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>researchTitle</tabstop>
  <tabstop>startDate</tabstop>
  <tabstop>endDate</tabstop>
  <tabstop>researchInfo</tabstop>
  <tabstop>editResearchersButton</tabstop>
  <tabstop>newPointButton</tabstop>
  <tabstop>editPointButton</tabstop>
  <tabstop>deletePointButton</tabstop>
  <tabstop>checklist</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
